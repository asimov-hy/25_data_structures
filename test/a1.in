Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris nec venenatis ligula. Etiam elit neque, tempus nec enim eu, tincidunt rutrum turpis. Fusce in ex nec tellus placerat convallis. Nam id condimentum diam, ac commodo dui. Vivamus at cursus erat, vitae elementum arcu. Maecenas congue dolor mauris, eu congue magna vehicula a. Donec efficitur, mi tincidunt dictum sagittis, orci tortor finibus massa, sit amet consequat risus libero eu mi. Curabitur a sapien neque. Nam ut nulla vel metus sagittis rhoncus in eu metus. Aliquam ex lectus, elementum non luctus vel, imperdiet condimentum nunc. Nulla bibendum mauris tellus, in volutpat lacus hendrerit in. Fusce in efficitur elit. Maecenas dolor massa, tempus in tellus eu, tempor fringilla erat. Morbi tempus urna auctor tempor tincidunt. Quisque non tortor quis ipsum dapibus placerat. Sed interdum dui a lacus porta interdum.