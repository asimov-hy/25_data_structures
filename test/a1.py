## Problem a1: String reverse


def reverse(input_data):
    return input_data[::-1]

# def reverse(input_data):
#     return input_data[::-1]