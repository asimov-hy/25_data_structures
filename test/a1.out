.mudretni atrop sucal a iud mudretni deS .tarecalp subipad muspi siuq rotrot non euqsiuQ .tnudicnit ropmet rotcua anru supmet ibroM .tare allignirf ropmet ,ue sullet ni supmet ,assam rolod saneceaM .tile ruticiffe ni ecsuF .ni tirerdneh sucal taptulov ni ,sullet siruam mudnebib alluN .cnun mutnemidnoc teidrepmi ,lev sutcul non mutnemele ,sutcel xe mauqilA .sutem ue ni sucnohr sittigas sutem lev allun tu maN .euqen neipas a rutibaruC .im ue orebil susir tauqesnoc tema tis ,assam subinif rotrot icro ,sittigas mutcid tnudicnit im ,ruticiffe cenoD .a alucihev angam eugnoc ue ,siruam rolod eugnoc saneceaM .ucra mutnemele eativ ,tare susruc ta sumaviV .iud odommoc ca ,maid mutnemidnoc di maN .sillavnoc tarecalp sullet cen xe ni ecsuF .siprut murtur tnudicnit ,ue mine cen supmet ,euqen tile maitE .alugil sitanenev cen siruaM .tile gnicsipida rutetcesnoc ,tema tis rolod muspi meroL